src/routing_membrane.rs

```diff
@@ -1326,6 +1363,16 @@ impl<F> RoutingMembrane<F> where F: Interface {
         Ok(())
     }
 
+    fn group_pub_keys(&self) -> BTreeMap<NameType, sign::PublicKey> {
+        let name_and_key_from_info = |node_info : NodeInfo| {
+            (node_info.fob.name(), node_info.fob.signing_public_key())
+        };
+
+        self.routing_table.target_nodes(&self.id.name())
```

Comment:
we need to use explicitly RT::our_close_group and add our own name and key to it